adobe/spectrum-web-components
# fix: ensure Action Menu Item with [href] close the menu

## Description
When an Action Menu is full of Menu Items with `href`s, it will now close after activating one of its Menu Items.
- updated Menu and Picker
- refactors Action Menu tests for sharing and adds two new tests

## Motivation and context
- Action Menu should be more active.

## How has this been tested?
    1. The tests pass.
    1. Go [here](https://action-menu-links--spectrum-web-components.netlify.app/storybook/index.html?path=/story/action-menu--icon-only)
    2. Add an `href` to one of the demo `<sp-menu-item>` elements.
    3. Open the Action Menu
    4. Command + click the Menu Item with an `href`
    5. See a new tab open
    6. See the Action Menu close
    7. profit.

## Types of changes

## Checklist
